Makefile

```
@@ -156,6 +156,12 @@ e2e-upgrade: $(GINKGO) $(KUSTOMIZE) $(KIND) $(GOVC) ## Run e2e tests
 	@echo
 
 	time $(GINKGO) -v -focus="ClusterAPI Upgrade Tests" ./test/e2e -- --e2e.config="$(E2E_CONF_FILE)"
+
+.PHONY: test-cover
+test-cover: ## Run tests with code coverage and code generate  reports
+	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v -coverprofile=coverage.out ./... $(TEST_ARGS)
```

This will run the E2E tests, something we wouldn't wanna run for this. Add
```suggestion
	$(MAKE) generate
	source ./hack/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v -coverprofile=coverage.out ./api/... ./controllers/... ./pkg/... $(TEST_ARGS)
```

Also where is `TEST_ARGS` defined?

---

I guess `TEST_ARGS` are the arguments to be provided while executing `make test-cover`, if any.